### ISSUE TITLE
The quickstrike dagger does no damage

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#285]
```
There is no difference between the damage done by the quickstrike dagger and 
your bare hands.

What steps will reproduce the problem?
1. Look at the description of the quickstrike, notice that it has no Attack 
damage field.
2.Take off all your items except for the quickstrike dagger.
3.Notice in your inventory screen that the damage you get with the dagger is 
exactly the same as your base damage.

What is the expected output? What do you see instead?
I would expect the dagger to have attack damage of 1-2 like other daggers in 
the game.


What version of the product are you using? On what device?

v0.6.10 on Samsung Galaxy GT9000

This is also mentioned in bug 88, which is an enhancement proposal for shops.  
The filer of that issue thought that there was a bug that the damage of the 
dagger was not _listed_ in the store, but the real problem is that the dagger 
doesn't have any damage at all, to list.  I considered adding this note to bug 
88, but decided that the problem was distinct enough from shop enhancement to 
deserve its own issue, especially since I suspect it is a 2 line initialisation 
fix.

Thank you for many happy hours playing on my phone.

```

Original issue reported on code.google.com by `160...@gmail.com` on 30 Nov 2011 at 8:25

**Comment by GoogleCodeExporter** [comment]
```
IMO its a invalid bug. Quickstrike dagger is far too overpowered. No need to 
make it stronger. There is no rule that a dagger must have AD.
```

Original comment by `SamuelPl...@gmail.com` on 30 Nov 2011 at 11:54

**Comment by GoogleCodeExporter** [comment]
```
I think that if this is the desired behaviour, then the quickstrike dagger 
should have a listed AD of 0.  But if the quickstrike dagger is overpowered, 
then I do not believe that the problem with it is damage-related. Right now you 
can get the exact same damage effect, and number of attacks per round, by 
wielding no weapon at all, so unless one thinks that bare hands are also 
overpowered, it is hard for me to see that unbalancing aspects of the 
quickstrike dagger have to do with it's damage potential. By making it your 
weapon of choice these days you are trading 20% in block chance for an extra 
20% of attack chance.  If that is something that is considered overpowered, 
then the solution is to change those values.
```

Original comment by `160...@gmail.com` on 1 Dec 2011 at 12:16

**Comment by GoogleCodeExporter** [comment]
```
> I think that if this is the desired behaviour, then the quickstrike dagger 
should have a listed AD of 0.

No. The current way to show a stat with value 0 is omitting it from the list.


> ...so unless one thinks that bare hands are also overpowered...

Correct. Bare hands are overpowered too.
See forum for endless discussions about overpowered bare hands and quickstrike 
dagger.
```

Original comment by `SamuelPl...@gmail.com` on 1 Dec 2011 at 1:38

**Comment by GoogleCodeExporter** [comment]
```
Fixed with barehand changes in v0.6.12
```

Original comment by `sdeva...@gmail.com` on 9 Feb 2013 at 5:55
- Changed state: **Fixed**
